index.html

```diff
@@ -7,12 +7,13 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap"
       rel="stylesheet"
-    />
+    /> <!-- hard to say best style-->
     <link
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
-    <link href="style.css" rel="stylesheet" />
```

Comment:
Is there a reason you have '/>' instead of just '>' on self-closing tags/void elements?